### PR TITLE
[ENG-2858] Remove old Travis references

### PR DESCRIPTION
**Pull request by jh27539**
## Purpose
Remove old references to Travis CI in our code base. 

## Summary of Changes
- Update github actions workflows
- Update init file
- Update README status badges for `Master` and `Develop`
- Combine 2 test partitions into a singular test run

## Reviewer's Actions
N/A

Run this test using
N/A

## Testing Changes Moving Forward
N/A

## Ticket

https://openscience.atlassian.net/browse/ENG-2858

### CODE REVIEW

File: tasks/__init__.py [hunk 1]
```diff
@@ -75,33 +75,33 @@ def test_module(ctx, module=None, params=None):
...
     print('>>> Testing modules in "{}" in Chrome'.format('tests'))
-    test_travis_with_retries(
+    test_selenium_with_retries(
         ctx, 'Master', _get_test_file_list(), module=['-m', 'smoke_test']
     )
 
 
 @task
-def test_travis_part_one(ctx):
+def test_selenium_part_one(ctx):
```
Comment (DougCorell) [inline tasks/__init__.py]: Do we still need to break the Staging tests into 2 parts? I thought that was only due to Travis limitations.

File: tasks/__init__.py [hunk 2]
```diff
@@ -75,63 +75,41 @@ def test_module(ctx, module=None, params=None):
...
-
 def _get_test_file_list():
     all_test_files = glob.glob('tests/test_*.py')
     all_test_files.sort()
     return all_test_files
 
 
 @task
-def test_travis_with_retries(ctx, partition_name, file_list, module=None):
+def test_selenium_with_retries(ctx, file_list, module=None):
```
Comment (DougCorell) [inline tasks/__init__.py]: Where is the file_list parameter being populated now when you are directly invoking this function from the selenium_staging_tests.yml file?

Comment (felliott) [reply tasks/__init__.py]: :+1:

File: tasks/__init__.py [hunk 3]
```diff
@@ -81,9 +81,7 @@ def test_selenium_on_prod(ctx):
     """
     flake(ctx)
     print('>>> Testing modules in "{}" in Chrome'.format('tests'))
-    test_selenium_with_retries(
-        ctx, 'Master', _get_test_file_list(), module=['-m', 'smoke_test']
-    )
+    test_selenium_with_retries(ctx, 'Master', module=['-m', 'smoke_test'])
```
Comment (jh27539) [inline tasks/__init__.py:84]: Update production run to use the same code.

File: tasks/__init__.py [hunk 4]
```diff
@@ -93,9 +91,10 @@ def _get_test_file_list():
 
 
 @task
-def test_selenium_with_retries(ctx, file_list, module=None):
+def test_selenium_with_retries(ctx, module=None):
     """Run group of tests on the browser defined by TEST_BUILD."""
     flake(ctx)
+    file_list = _get_test_file_list()
```
Comment (jh27539) [inline tasks/__init__.py:97]: Grab the file list at the beginning of this method since we are no longer passing it in.

**Pull request closed without merging**